README.md

```diff
@@ -46,9 +78,32 @@ The versions follow a pattern `MAJOR.MINOR.PATCH.STUB_VERSION` where the first t
 
 ## Type checkers
 
-As of now Mypy is the only type checker the stubs were tested with. 
+As of now Mypy is the only type checker the stubs were tested with.
```

Comment:
Sometimes you use small letter for mypy and sometimes capital.

Reply:
Fixed. I'll use the capital letter only at the beginning of sentences.